pennylane/tape/qscript.py

```diff
@@ -1134,21 +1129,10 @@ def adjoint(self):
         """
         with qml.QueuingManager.stop_recording():
             ops_adj = [qml.adjoint(op, lazy=False) for op in reversed(self._ops)]
-        adj = self.__class__(
+        return self.__class__(
```

Comment:
Should this be added to the queue if `QueuingManager.recording()` is `True`? Or is this now taken care of in pennylane/tape/tape.py in line 378 and 387?

Reply:
I think this is the simplest way. We don't really want to add scripts to queues, only operators/measurements. The exception is backwards-compatibility for `QuantumTape` specifically, so I'm intentionally adding the support there.